packages/lit-dev-content/samples/tutorials/working-with-lists/03/after/my-element.ts

```diff
@@ -1,5 +1,6 @@
 import {LitElement, html} from 'lit';
 import {customElement, property, state} from 'lit/decorators.js';
+import type {TemplateResult} from 'lit';
```

Comment:
This type import and the `const listItems: TemplateResult[] = [];` initialization need to be updated in the `before` file.

Reply:
Oops, thanks for catching that! It's weird that prior to this TS was happily inferring the type for the array. I guess it has trouble following what happens inside `.forEach`'s callback.